src/tests/globalStyles.e2e.ts

```diff
@@ -38,17 +38,17 @@ describe("global styles", () => {
       });
     });
 
-    it("should set animation duration to 0ms when --animation-timing-factor set to zero", async () => {
+    it("should set animation duration to 0ms when --calcite-duration-factor set to zero", async () => {
       const page = await newE2EPage({
         html: html`
           <html>
             <style>
               html {
-                --calcite-animation-timing-factor: 0;
+                --calcite-duration: 0;
```

Comment:
This should be `--calcite-duration-factor`, right? This also applies to the next test.

Reply:
Yup.